.github/workflows/node-flow-fsts-daily-interval-02.yaml

```diff
@@ -102,7 +102,7 @@ jobs:
       grafana-agent-password: ${{ secrets.GRAFANA_AGENT_PASSWORD }}
     needs:
       - crypto-migration-7n-1c
-    if: ${{ !cancelled() && always() }}
+    if: ${{ !cancelled() && always() && !github.event.workflow_dispatch.repository.fork }}
```

Comment:
Does not work since `always()` always returns `true` plus has some additional behaviors. Need to change the ordering.